README.md

```diff
@@ -8,26 +8,29 @@ Endless Sky is a sandbox-style space exploration game similar to Elite, Escape V
 
 ## System Requirements
+
 Endless Sky has very minimal system requirements, meaning most systems should be able to run the game. The most restrictive requirement is likely that your device must support at least OpenGL 3.
 
 || Minimum | Recommended |
 |---|----:|----:|
 |RAM | 500 MB | 1 GB |
 |Graphics | OpenGL 3.0 | OpenGL 3.3 |
-|Storage Free | 250 MB | 1 GB |
+|Storage Free | 300 MB | 1 GB |
 
 ## Building from source
+
 Most development is done on Linux and Windows, using CMake ([build instructions](readme-cmake.md)) to compile the project. For those wishing to use an IDE, project files are provided for [Code::Blocks](https://www.codeblocks.org/) to simplify the project setup, and other IDEs are supported through their respective CMake integration. [SCons](https://scons.org/) was the primary build tool up until 0.9.16, and some files and information continue to be available for it.
 For full installation instructions, consult the [Build Instructions](https://github.com/endless-sky/endless-sky/blob/master/readme-developer.md) readme.
 
-# Contributing
-As a free and open source game, Endless Sky is the product of many peoples' work. Contributions of artwork, storylines, and other writing are most in-demand, though there is a loosely defined [roadmap](https://github.com/endless-sky/endless-sky/wiki/DevelopmentRoadmap). Those who wish to [contribute](CONTRIBUTING.md) are encouraged to review the [wiki](https://github.com/endless-sky/endless-sky/wiki), and to post in the [community-run discord](https://discord.gg/ZeuASSx) beforehand. Those who prefer to use Steam can use its [discussion rooms](https://steamcommunity.com/app/404410/discussions/) as well. For those who do not wish to use either discord or steam, we do have access to [github discussions](https://github.com/endless-sky/endless-sky/discussions), although these are quite new and don't have a lot of traffic yet.
+## Contributing
+
+As a free and open source game, Endless Sky is the product of many peoples' work. Contributions of artwork, storylines, and other writing are most in-demand, though there is a loosely defined [roadmap](https://github.com/endless-sky/endless-sky/wiki/DevelopmentRoadmap). Those who wish to [contribute](CONTRIBUTING.md) are encouraged to review the [wiki](https://github.com/endless-sky/endless-sky/wiki), and to post in the [community-run Discord](https://discord.gg/ZeuASSx) beforehand. Those who prefer to use Steam can use its [discussion rooms](https://steamcommunity.com/app/404410/discussions/) as well, or GitHub's [discussion zone](https://github.com/endless-sky/endless-sky/discussions).
 
-Endless Sky's main discussion and development area was once [Google Groups](https://groups.google.com/g/endless-sky), but due to factors outside our control, it is now inaccessible to new users.
```

Comment:
Makes sense. I've added a small bit to the end that makes it a bit more clear IMO. Thoughts?

Reply:
> I think it's worth keeping the link.

What value does the link have?

Reply:
Have you tried visiting the Google groups forum recently?
I read many of the posts there back when it was still usable, and I can't even get at it now.
I'm not sure how anyone who doesn't already know about it is going to be able to make any use of it. Any mention of it here will likely just be confusing, certainly not useful.

Reply:
Congratulations! You're not in the "doesn't already know about it" group, and so don't need the readme to tell you about it.

Reply:
I consider it more a matter of honesty and transparency.

Reply:
Hardly transparent when very few people can access it.

Reply:
IMHO deleting the link wouldn't be pretending it never existed, just moving the reference into the commit history.

Reply:
Commit histories are useful for looking up past states of things or tracking the development of something. They are also utterly unknown to most people.

Reply:
> They are also utterly unknown to most people.

Just like the Google groups then? At least the commit histories are visible to anyone with access to the internet.

Reply:
OK. Just take a step back here. Why are we arguing about a single one-line reference that acknowledges that someplace was where significant development discussion took place?

- No one is being misled by this. It's very clear, and quyykk's change makes it even clearer.
- It maintains honesty and transparency about our development history.
- It provides an explanation so that if anyone (say, past people who played) comes looking for what happened to the google groups, there's a clear and easy to find.
- It doesn't harm anyone or anything.
- It takes up virtually zero space.

Reply:
I cannot access Google Groups at all.

Reply:
Based on past research (over a year ago), the only people who can access the google groups now are people who joined it a bunch of years ago, and they can only do so on the account they originally joined it with.

Reply:
I think leaving the link there and say it's defunct is fine. We still got people returning from 5+ years ago who may still have access to that group and want to look for something. Leaving the link there would be easier for them as browser history most likely don't go that far (assuming they're still using the same machine).

Reply:
I'm fine with leaving the link.